### PR TITLE
Move examples of Embed Targets to the Appendix

### PR DESCRIPTION
**Pull request by csuwildcat**

### CODE REVIEW

**Review by glcohen — CHANGES_REQUESTED** [review]
this includes the steps pr and should not

**Comment by csuwildcat** [comment]
Should be all set now, take a look!

File: spec/spec.md [hunk 1]
```diff
@@ -61,16 +66,19 @@ work is being done.
...
-Holder | The entity that submits proofs to a Verifier to satisfy the requirements described in a Presentation Definition
-Verifier | The entity that defines what proofs they require from a Holder (via a Presentation Definition) in order to proceed with an interaction.
+[[def:Decentralized Identifiers, Decentralized Identifiers, DID]]
+~ Unique ID URI string and PKI metadata document format for describing the cryptographic keys and other fundamental PKI values linked to a unique, user-controlled, self-sovereign identifier in a target system (i.e. blockchain, distributed ledger).
+
+[[def:Holder, Holders]]
+~ The entity that submits proofs to a [[ref:Verifier]] to satisfy the requirements described in a Presentation Definition
+
+[[def:Verifier, Verifiers]]
+~ The entity that defines what proofs they require from a [[ref:Holder]] (via a Presentation Definition) in order to proceed with an interaction.
```
Comment (glcohen) [inline spec/spec.md:76]: What presentations?

**Review by glcohen — APPROVED** [review]
Might change after the terminology section. I think proofs is overloaded. Looks good

File: spec/spec.md (hunk 1 above)
Comment (csuwildcat) [reply spec/spec.md:76]: Not sure, but I didn't change any of the definitions, I just put them into the format that will auto-link them. Let's address the contents of the terms in another PR. (I am sure they could use work, in addition incorporating all the other terms we identified). This was more to get the format in so people could party on adding/modding terms.

**Pull request merged**